modules/search/instant-search/components/search-result-product.jsx

```diff
@@ -0,0 +1,62 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { h, Component } from 'preact';
+
+/**
+ * Internal dependencies
+ */
+import SearchResultComments from './search-result-comments';
+
+class SearchResultProduct extends Component {
+	render() {
+		const { result_type, fields, highlight } = this.props.result;
+		if ( result_type !== 'post' ) {
+			return null;
+		}
+
+		const firstImage = Array.isArray( fields[ 'image.url.raw' ] )
+			? fields[ 'image.url.raw' ][ 0 ]
+			: fields[ 'image.url.raw' ];
+
+		return (
+			<li className="jetpack-instant-search__search-result-product">
+				<h3>
+					<a
+						href={ `//${ fields[ 'permalink.url.raw' ] }` }
+						target="_blank"
+						rel="noopener noreferrer"
+						className="jetpack-instant-search__result-product-title"
+						//eslint-disable-next-line react/no-danger
+						dangerouslySetInnerHTML={ { __html: highlight.title } }
+					/>
+				</h3>
+				{ firstImage && (
+					<img
+						className="jetpack-instant-search__search-result-product-img"
+						src={ `//${ firstImage }` }
+						alt=""
```

Comment:
Something to note for the future: We should provide a proper alt text here for a11y reasons.

Reply:
The only information we have about the image from the API is the product title, so an alt here would be duplicating the title immediately above it. I _think_ that meets the definition of "decorative image":

https://www.w3.org/WAI/tutorials/images/decorative/

> Illustrative of adjacent text but not contributing information (“eye-candy”);
> Identified and described by surrounding text.